### DISCUSSION TITLE
Updated template for platform.sh (?)

### DISCUSSION THREAD

**Discussion by cosminepureanu** [directus/directus#6512] [Archive]
The [current template](https://github.com/platformsh-templates/directus) for platform.sh works perfectly. However, it uses the version `9.0.0-rc.31` of Directus. When I change the version to `9.0.0-rc.73` and deploy to platform.sh, the script stops with these lines of log:

```
Preparing deployment
      Closing services app and router
      Opening application app and its relationships
      Executing deploy hook for application app
        W: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by /app/node_modules/argon2/lib/binding/napi-v3/argon2.node)
```

I noticed that Directus requires `node >= 16` but platform.sh supports node v14 max. Could this be the problem? Is there a way to make the latest version of Directus available to platform.sh?

**Comment by rijkvanzanten** [comment]
> I noticed that Directus requires node >= 16

This isn't true. Directus required Node 12+. You only need Node 16 when you run Directus in development mode (for contributing 👍🏻 )

**Reply by cosminepureanu** [reply]
All right, I see. Thanks, I'll investigate further then. 👍

**Reply by rijkvanzanten** [reply]
That error suggests that the version of linux that Platform.sh uses for this is using an older version of GLIBC, that is incompatible with the argon2 library 🤔
